.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +11,4 @@
 - [ ] Added or updated documentation
 - [ ] Tests for new functionality and regression tests for bug fixes
 - [ ] Screenshots attached (for UI changes)
+- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
```

Comment:
Wanted to add this here, so that there's a handy link to the contribution section in each PR.